src/math/p5.Vector.js

```diff
@@ -1989,20 +1989,20 @@ p5.Vector.random3D = function random3D() {
  * @static
  * @param  {p5.Vector} v1 a <a href="#/p5.Vector">p5.Vector</a> to add
  * @param  {p5.Vector} v2 a <a href="#/p5.Vector">p5.Vector</a> to add
- * @param  {p5.Vector} target the vector to receive the result
- */
-/**
- * @method add
- * @static
- * @param  {p5.Vector} v1
- * @param  {p5.Vector} v2
```

Comment:
why was this signature removed?

Reply:
I remove all of these to make target value [target] (optional) as the functions are always returning resultant vector